app/pathogen/arbovirus/wip/sections.tsx

```diff
@@ -1,76 +1,5 @@
-import { RefObject } from "react";
-import { ArbovirusMap } from "../dashboard/(map)/ArbovirusMap";
-import { ArboDataTable } from "@/app/pathogen/arbovirus/analyze/ArboDataTable";
-import { VisualizationId, addToVisualizationInformation } from "../visualizations/visualizations";
-import { ZoomIn } from "lucide-react";
-import { useRouter } from "next/navigation";
-import { VisualizationsSection } from "./visualizations-section";
-import { RenderScrollSectionContentFunction } from "@/components/ui/scroll-section-group/scroll-section-group";
-
 export enum RedesignedArbovirusPageSectionId {
```

Comment:
Can't put this in the same file as the page because nextjs is expecting only one thing exported from a page. If I could do that I would delete this tiny file but I can't so here it is lol

Reply:
That is ok, when we have more pages we will want our sections to exist all in one place so we can control it better. This will allow us to create objects that have deeper nesting and scrollable links if needed in the future. 

I will say maybe it would be better for this file to live elsewhere in the application (utils or constants)

Additionally could change it to ts file (not sure the impact of this but I think it is good practice)

Reply:
I think a `constants.ts` is a pretty reasonable approach so that's what I've chosen to send this to. It's at the base of the `app` folder which feels natural to me but if we want it moved somewhere else just let me know and I'll move it hahaha

Also yeah no impact on making it a `.ts` file, that sounds good to me and I did it as a part of this commit.

https://github.com/serotracker/dashboard/pull/183/commits/db5dcdd1fbed7f7006e387e6b78668e87d79a2a7